doc/examples/applications/plot_text.py

```diff
@@ -16,9 +16,9 @@
 
 img = data.cat()
 
-fig = plt.figure()
-fig.figimage(img, resize=True)
-fig.text(0, 0.99, "I am stefan's cat.", fontsize=32, va="top")
+fig, ax = plt.subplots()
+ax.figure.figimage(img, resize=True)
+ax.text(0, 0.99, "I am stefan's cat.", fontsize=32, va="top")
```

Comment:
Something in these changes apparently breaks the [example](https://output.circle-artifacts.com/output/job/7542931b-2ec6-4606-a94e-e33e10db810f/artifacts/0/doc/build/html/auto_examples/applications/plot_text.html)? There's not text on the image?

Probably because `text()` is now called on `ax` instead. Maybe just revert since `fig = plt.figure()` seems like OO?

```suggestion
fig = plt.figure()
fig.figimage(img, resize=True)
fig.text(0, 0.99, "I am stefan's cat.", fontsize=32, va="top")
```